### ISSUE TITLE
Increase touch target for the play button in tabs tray - grid layout

### ISSUE THREAD

**Issue by Mugurell** [mozilla-mobile/fenix#15971]
Followup ticket for https://github.com/mozilla-mobile/fenix/pull/15970.

The items is still seen by the a11y scanner as having a 24x24 size though [playPauseButtonView.increaseTapArea](https://github.com/mozilla-mobile/fenix/blob/5bd829f389ad439719642baaf6a70b909df10e6f/app/src/main/java/org/mozilla/fenix/tabtray/TabTrayViewHolder.kt#L97) is already being used.

We need to have this button have at least 48dp in size, while staying with the current design.

**Comment by mcarare** [comment]
As a note, the Accessibility scanner does not account for the use of `TouchDelegate` for versions before Android 10. If using the scanner on an Android 10 device it does not show any suggestions, the layout should be fine without any additional changes.

**Comment by Mugurell** [comment]
I think the issue here is that according to [the current design](https://www.figma.com/file/luMaQYjwiMtQM5xQ30P9yP/Tabs?node-id=168%3A713) there is no space for where the button's touch rect to extend.
The button is already to the extreme corner of the layout.
<img src="https://user-images.githubusercontent.com/11428869/96640782-604bc800-132c-11eb-83a0-f6c67217b80b.png" width="400"/>

Waiting for UX feedback on how to resolve this:
- increase the spacing between items from 16dp to 28dp to allow the button to extend from 24dp to 48dp
- move the button lower in the layout by 12 dp to allow the same size increase
- other

**Comment by topotropic** [comment]
@Mugurell I assume this means the play button needs to be centered in the 48x48dp parent box, correct? Or could we position the play button in the uper left corner and let the parent box extend to the right and bottom?

**Comment by Mugurell** [comment]
Maybe a bit strange but I think after the changes from https://github.com/mozilla-mobile/fenix/pull/16481
specifically the removal of this line - https://github.com/mozilla-mobile/fenix/blob/a8db85fc22cbdb5ae36638e24d3892c0a186ee52/app/src/main/res/layout/tab_tray_grid_item.xml#L21
the behavior will be the one you talked about Nicole - the 48x48 touch rect parent box will be translated to fit the parent.
The play button will not be centered but in a corner of that touch rect which will no be big enough to not be an a11y issue.

This should be be rechecked after that PR lands.

**Comment by sflorean** [comment]
@Mugurell this is still reproducible on Nightly 11/17 on LG G7 Fit (Android 8.0). A suggestion is made to increase the size from 24dp x 24dp to 48dp.

**Comment by mcarare** [comment]
> As a note, the Accessibility scanner does not account for the use of `TouchDelegate` for versions before Android 10

https://support.google.com/accessibility/android/answer/7101858?hl=en

>Note: Accessibility Scanner can detect and account for the use of TouchDelegate only when running on Android 10 and later. On earlier Android versions, touch target size results may appear even when this API is used to enlarge touch targets to an appropriate size.

**Comment by sflorean** [comment]
Based on the [comment](https://github.com/mozilla-mobile/fenix/issues/15971#issuecomment-729544808) above, I will close this issue.
